cmd/controller/app/options/options.go

```diff
@@ -18,8 +19,9 @@ type ControllerOptions struct {
 }
 
 const (
-	defaultAPIServerHost = ""
-	defaultNamespace     = ""
+	defaultAPIServerHost            = ""
+	defaultNamespace                = ""
+	defaultClusterResourceNamespace = "kube-system"
```

Comment:
Can we default to the current namespace instead? e.g. `/var/run/secrets/kubernetes.io/serviceaccount/namespace`

Reply:
Just remembered this comment after merging. I've opened https://github.com/jetstack-experimental/cert-manager/issues/103